openbb_terminal/api.py

```diff
@@ -62,59 +98,131 @@ def __call__(self, *args, **kwargs):
             return self.model(*args, **kwargs)
 
 
-
 class Item:
-    def __init__(self, function):
+    def __init__(self, function: Callable):
         self.__function = function
 
     def __call__(self, *args, **kwargs):
         self.__function(*args, **kwargs)
 
 
 class APILoader:
+    """The APILoader class"""
     def __init__(self, shortcuts: dict):
         self.__mapping = self.build_mapping(shortcuts=shortcuts)
         self.load_items()
 
+    def __call__(self):
+        """Prints help message"""
+        print("""This is the API of the OpenBB Terminal.
+        
+        Use the API to get data directly into your jupyter notebook or directly use it in your application.
+        For documentation use: 
+        - help(<api>.<menu>.<command>.<model or view>)
+        - or: <api>.<menu>.<command>.<model or view>.__doc__
+        
+        Use <api>.settings() to change the settings of the api instance (for example: openbb.settings(bla bla)
+        ...
+        
+        For more information see the official documentation at: https://openbb-finance.github.io/OpenBBTerminal/api/
+        """)
+
+    # TODO: Add settings
+    def settings(self):
+        pass
+
     def load_items(self):
+        """Creates the API structure (see api.stocks.command) by setting attributes and saving the functions"""
         mapping = self.__mapping
-        for shorcut, function in mapping.items():
-            shortcut_split = shorcut.split(".")
-            last_shortcurt = shortcut_split[-1]
+        for shortcut, function in mapping.items():
+            shortcut_split = shortcut.split(".")
+            last_shortcut = shortcut_split[-1]
 
             previous = self
             for item in shortcut_split[:-1]:
                 next_item = Item(function=item)
                 previous.__setattr__(item, next_item)
                 previous = next_item
 
-            previous.__setattr__(last_shortcurt, function)
+            previous.__setattr__(last_shortcut, function)
+
+    @staticmethod
+    def load_module(module_path: str) -> Optional[types.ModuleType]:
+        """Load a module from a path.
+        Args:
+            module_path (str):
+                Module"s path.
+        Returns:
+            Optional[ModuleType]:
+                Loaded module or None.
+        """
+
+        try:
+            spec = importlib.util.find_spec(module_path)
+        except ModuleNotFoundError:
+            spec = None
+
+        if spec is None:
+            return None
+        else:
+            module = importlib.import_module(module_path)
+
+            return module
 
     @staticmethod
```

Comment:
If you need the object attributes/methods it's a normal method.
If you need to use the class properties/functions inside the function it's better to turn it into a `@classmethod`. 
If you don't need anything related to the object or the class you can turn it into a `@staticmethod`.

Here I would turn it into a `@classmethod`.

Reply:
Done!